content/bn/cloud_native_tech.md

```diff
@@ -0,0 +1,23 @@
+---
+title: ক্লাউড নেটিভ প্রযুক্তি
+status: সম্পন্ন
```

Comment:
```suggestion
status: Completed
```
Please do not translate `status`. It will be used as a Keyword for building site.
(FYI: If `status: Completed` not, you will not find title of this term in side-bar tree)

Reply:
I have updated that